### PR TITLE
Fix dynamic rnn comment

### PR DESCRIPTION
**Pull request by pengwa**

### CODE REVIEW

File: tensorflow/python/ops/rnn.py [hunk 1]
```diff
@@ -709,6 +709,10 @@ def _dynamic_rnn_loop(cell,
   Raises:
     ValueError: If the input depth cannot be inferred via shape inference
       from the inputs.
+    ValueError: If time_step is not the same for all the elements in the
+      input.
```
Comment (yifeif) [inline tensorflow/python/ops/rnn.py]: super nit, input -> inputs.

**Review by yifeif — CHANGES_REQUESTED** [review]

File: tensorflow/python/ops/rnn.py (hunk 1 above)
Comment (pengwa) [reply tensorflow/python/ops/rnn.py]: pl

**Comment by pengwa** [comment]
@yifeif have fixed as you suggested. please help check.

**Review by yifeif — APPROVED** [review]
Thanks @pengwa!

**Pull request merged**